File: Bimestre 2/Java/Taller2B_1/src/taller2b_1/Ejercicio1.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package taller2b_1;

import java.util.Scanner;

/**
 *
 * @author diegojp
 */
public class Ejercicio1 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner leer = new Scanner(System.in);
        
        int n,i,aux,Ai,Bi;
        n=5;aux=0;Ai=0;Bi=0;
        int [] A = new int [n];
        int [] B = new int [n];
        for(i=0;i<n;i++){
            System.out.println("Ingresar los 5 Primeros Numeros del Vector A");
            A[i]=leer.nextInt();
            
        }
        System.out.println("--------------------------------------------");
        for(i=0;i<n;i++){
            System.out.println("Ingresar los 5 Primeros Numeros del Vector B");
            B[i]=leer.nextInt();
        }
        System.out.println("--------------------------------------------");
        System.out.println("***Vector original A *******");
        for(i=0; i<n; i++){
            
            System.out.println("A["+i+"]="+A[i]);

        }
        System.out.println("********Vector original B*********");
        for(i=0; i<n; i++){
            
            System.out.println("B["+i+"]="+B[i]);
        }

        for(i=0;i<n;i++){
            if((A[i]==B[i])&&(i==i)){
                System.out.println("El vector A y el vector B tienen el "
                        + "mismo numero "+A[i]+" y comparten la misma posicion en ["+i+"]");
            }
        }
    }
    
    
}

File: Bimestre 2/Python/taller2b_1/Ejercicio1.py

n=5;aux=0;Ai=0;Bi=0;
A=[]
B=[]
for i in range(0, n):
    print("Ingresar los 5 Primeros Numeros del Vector A")
    val=int(input())
    A.append(val)

print("--------------------------------------------")
for i in range(0, n):
    print("Ingresar los 5 Primeros Numeros del Vector B")
    val=int(input())
    B.append(val)

print("--------------------------------------------")
print("***Vector original A *******")
for i in range(0, n):
    print("A[",i,"]=",A[i], end=" ")

print("********Vector original B*********")
for i in range(0, n):
    print("B[",i,"]=",B[i], end=" ")

for i in range(0, n):
    if((A[i]==B[i])and(i==i)):
        print("El vector A y el vector B tienen el mismo numero ",A[i]," y comparten la misma posicion en [",i,"]")
